core-bundle/src/Resources/contao/widgets/FileTree.php

```diff
@@ -386,6 +386,7 @@ public function generate()
                 $("ctrl_' . $this->strId . '").getParent("div").set("html", json.content);
                 json.javascript && Browser.exec(json.javascript);
                 $("ctrl_' . $this->strId . '").fireEvent("change");
+                ' . ($this->arrConfiguration['submitOnChange'] ? 'Backend.autoSubmit("' . $this->strTable . '");' : '') . '
```

Comment:
```suggestion
                $("ctrl_' . $this->strId . '").fireEvent("change");' . ($this->arrConfiguration['submitOnChange'] ? '
                Backend.autoSubmit("' . $this->strTable . '");' : '') . '
```

Reply:
@AlexejKossmann It seems you have resolved this conversation without committing my suggestion or commenting why you do not want to commit it. 😄

Reply:
@leofeyer Sorry, i have overseen this change request. Too bad, i have already deleted this repository. 
Your changes are alright. Can you merge them without me, or do i need to check this commit out again?